Utilities/Config.cpp

```diff
@@ -136,6 +136,35 @@ bool try_to_uint64(u64* out, std::string_view value, u64 min, u64 max)
 	return true;
 }
 
+std::vector<std::string> cfg::make_float_range(f64 min, f64 max)
+{
+	return {std::to_string(min), std::to_string(max)};
+}
+
+bool try_to_float(f64* out, std::string_view value, f64 min, f64 max)
+{
+	f64 result;
+	const char* start = &value.front();
+	const char* end = &value.back() + 1;
```

Comment:
all of these need a return false if value.empty()

Reply:
Done.

Reply:
the reason being that `const char* end = &value.back() + 1;`
is a really dumb thing to to

Reply:
It's also a UB for front() which I changed.

Reply:
in fact some of these functions access the second character without ever checking if there is one